packages/http-client-java/Setup.ps1

```diff
@@ -1,4 +1,5 @@
 # re-build http-client-java
 npm install 
 npm run clean && npm run build
+npm run lint
```

Comment:
Add `npm run lint` here.

Reply:
Please add `npm run lint` after this `npm run build` line in [Build-Packages.ps1](https://github.com/microsoft/typespec/blob/main/packages/http-client-java/eng/scripts/Build-Packages.ps1#L75) script.